src/App.vue

```diff
@@ -98,6 +98,14 @@ export default {
     },
   },
   async created() {
+    EventBus.$on('redirectMaintenance', (err) => {
```

Comment:
Event names should represent what happened (not how it will be handled), so I guess this should be something like 'backendError'